worldedit-core/src/main/java/com/boydti/fawe/object/RelightMode.java

```diff
@@ -0,0 +1,7 @@
+package com.boydti.fawe.object;
+
+public enum RelightMode {
+    NONE,
+    OPTIMAL,
+    ALL
```

Comment:
There should be a comment here describing the differences between OPTIMAL and ALL. Anyone using the code in the future should know what OPTIMAL actually means.